descriptors/ModuleDescriptor-template.json

```diff
@@ -259,6 +259,16 @@
       "displayName": "Acquisition piece events - get piece change events",
       "description": "Get piece change events"
     },
+    {
+      "permissionName": "acquisition.piece.events.history.get",
+      "displayName": "Acquisition piece status change history events - get piece status change events",
+      "description": "Get piece status change events"
+    },
+    {
+      "permissionName": "pubsub.events.post",
```

Comment:
Hello, 
I see in mod-pubsub module readme.md file these lines mentioned below. Will it be fine to remove the pusub.events.post permission from the permissionRequired section and use a mod-audit permission ?

Subscriber module needs to provide endpoints for all subscriptions listed in the subscriptions block of its MessagingDescriptor.json. Let's add an interface to the subscriber's provides block in the ModuleDescriptor-template.json. **Note that each handler should have pubsub.events.post permission in the requiredPermissions which would allow mod-pubsub to call this endpoint.**
```
{
  "id": "{module-name}-event-handlers",
  "version": "{version}",
  "handlers": [
    {
      "methods": [
        "POST"
      ],
      "pathPattern": "/callback/address/example",
      "permissionsRequired": [
        "pubsub.events.post"
      ],
      "modulePermissions": [
        "required.permission.if.needed"
      ]
    }
  ]
}
```

Reply:
More details are provided in this Jira - https://folio-org.atlassian.net/browse/MODPUBSUB-145

Reply:
Hi @OleksiiKuzminov ,
As per the above comments and suggestions we believe that can Eureka team add an exception for "pubsub.events.post" to the validator. This permission cant be redeclared in mod-audit module as stated above in the comments.

Reply:
@TarasSpashchenko Can you please clarify, what is a final decision related to mod-pub-sub?

Reply:
https://folio-project.slack.com/archives/C210RP0T1/p1728648671681809?thread_ts=1728549680.865499&cid=C210RP0T1

Reply:
Please check Craig's response.